examples/example-tutorial-todo-app/src/Home.tsx

```diff
@@ -30,10 +30,10 @@ function Home() {
       <div className={css.tutorialBannerWrapper}>
         <div className={css.tutorialBanner}>
           <p className={css.tutorialBannerTitle}>
-            💡 Welcome to this tutorial!
+            💡 Welcome to To-do app tutorial!
           </p>
           <p>
-            The Todo App below has been implemented with fake in memory data.
+            The To-do App is implemented with mock in-memory data.
```

Comment:
Do we want to change all To-do  -> To Do to be consistent with the [AIP product](https://build.palantir.com/platform/85130d51-6134-43cf-995a-153777d83777). I forgot which one is preferred 😄 

```suggestion
            The To Do App is implemented with mock in-memory data.
```